nextflow.config

```diff
@@ -70,11 +70,11 @@ params {
     email                           = null
     email_on_fail                   = null
     plaintext_email                 = false
-    monochrome_logs                 = false
     hook_url                        = null
     help                            = false
     version                         = false
     validate_params                 = true
+    monochromeLogs                  = false
```

Comment:
what does this param do?

Reply:
It's to tell the nf-validation plugin if it should output all parameters at the start of the run in a nice color or just in the terminal color